.codeclimate.yml

```diff
@@ -3,6 +3,8 @@ engines:
     enabled: false
   eslint:
     enabled: true
+    exclude_paths:
+    - "test/**"
```

Comment:
ESLint was actually the one I thought _should_ be analyzing the files. 

I'm content with moving this back to the top-level excludes and thinking about it later